plugins/woocommerce/tests/e2e-pw/bin/test-helper-apis.php

```diff
@@ -136,3 +146,19 @@ function get_environment_info() {
 
 	return new WP_REST_Response( $data, 200 );
 }
+
+/**
+ * Activate a theme via the REST API.
+ * @param WP_REST_Request $request Request object.
+ * @return WP_REST_Response
+ */
+function activate_theme_via_api( WP_REST_Request $request ) {
+	$theme_name = sanitize_text_field( $request['theme_name'] );
+
+	if ( wp_get_theme( $theme_name )->exists() ) {
```

Comment:
Sending an empty string as theme_name or sending an invalid request body will "activate" the "" theme.

```
{
    "message": "Theme '' activated successfully."
}
```

Reply:
Sure I can add a check for that.